Failure on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.40608&attempt_id=2633c414-9e26-4a6f-b469-3d6dc2e4b931)

Attempted: iasl

<details><summary>Partial log (click to expand)</summary><p>

```
   return __builtin___sprintf_chk (__s, __USE_FORTIFY_LEVEL - 1,
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
       __bos (__s), __fmt, __va_arg_pack ());
       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors
make[1]: *** [../Makefile.rules:20: obj/aslanalyze.o] Error 1
make[1]: Leaving directory '/build/acpica-unix-20180508/generate/unix/iasl'
make: *** [generate/unix/Makefile.common:7: iasl] Error 2
builder for '/nix/store/f569i1cw3gw6qqbbc1vz1zafr4rhgggv-iasl-20180508.drv' failed with exit code 2
error: build of '/nix/store/f569i1cw3gw6qqbbc1vz1zafr4rhgggv-iasl-20180508.drv' failed
```
</p></details>

--- a/pkgs/development/compilers/iasl/default.nix
+++ b/pkgs/development/compilers/iasl/default.nix
@@ -9,7 +9,11 @@ stdenv.mkDerivation rec {
     sha256 = "1n7lqmv77kg28drahvxzybwl9v4hzwi8i7xkpgliclfcp5ff909b";
   };
 
-  NIX_CFLAGS_COMPILE = "-O3";
+  NIX_CFLAGS_COMPILE = [
+    "-O3"
+    # See: https://github.com/acpica/acpica/issues/387:
+    "-Wno-error=format-overflow"
+  ];
 
   buildFlags = "iasl";
 

